### PR TITLE
[node-bridge] add feature flags support

### PR DESCRIPTION
**Pull request by Kikobeats**
This PR allows executing conditional logic based in feature flags.

### CODE REVIEW

File: packages/node-bridge/bridge.js [hunk 1]
```diff
@@ -63,6 +64,7 @@ function normalizeProxyEvent(event) {
 
   if (payloads) {
     for (const p of payloads) {
+      p.features = payload.features;
```
Comment (styfle) [inline packages/node-bridge/bridge.js]:
```suggestion
      p.features = features;
```

Nit pick: this was destructured above so you can use the shorter value here

**Review by styfle — COMMENTED** [review]
Can you add a test?

**Comment by Kikobeats** [comment]
test added 🙏

File: packages/node-bridge/test/bridge.test.js [hunk 2]
```diff
@@ -29,21 +29,34 @@ test('`NowProxyEvent` normalizing', async () => {
...
         method: 'POST',
         headers: { foo: 'baz' },
+        features: { enabled: true },
         path: '/nowproxy',
         body: 'body=1',
       }),
     },
     context
   );
+  assert.strictEqual(features.enabled, true);
```
Comment (styfle) [inline packages/node-bridge/test/bridge.test.js]:
```suggestion
  assert.deepStrictEqual(features, { enabled: true });
```

Lets check the whole object

**Review by styfle — DISMISSED** [review]

**Review by TooTallNate — APPROVED** [review]

**Review by styfle — APPROVED** [review]

**Pull request merged**